<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgMessage</class>
 <widget class="QDialog" name="DlgMessage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>消息窗</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ssp_iaFloatView.qrc">
    <normaloff>:/images/information</normaloff>:/images/information</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_critical">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="ssp_iaFloatView.qrc">:/images/delete</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_warn">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="ssp_iaFloatView.qrc">:/images/bonus</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_information">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="ssp_iaFloatView.qrc">:/images/information</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_question">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="ssp_iaFloatView.qrc">:/images/help</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QPlainTextEdit" name="plainTextEdit"/>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>159</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btn_ok">
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>是(&amp;Y)</string>
        </property>
        <property name="icon">
         <iconset resource="ssp_iaFloatView.qrc">
          <normaloff>:/images/ok</normaloff>:/images/ok</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btn_cancel">
        <property name="text">
         <string>否(&amp;N)</string>
        </property>
        <property name="icon">
         <iconset resource="ssp_iaFloatView.qrc">
          <normaloff>:/images/cancel</normaloff>:/images/cancel</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ssp_iaFloatView.qrc"/>
 </resources>
 <connections/>
</ui>
